bots/tests-scan

```diff
@@ -49,7 +49,7 @@ NPM_REPOS = {
 REDHAT_VERIFY = {
     "verify/rhel-7": [ 'master' ],
     "verify/rhel-7-4": [ 'master', 'rhel-7.4' ],
-    "verify/rhel-7-5": [ ],
+    "verify/rhel-7-5": [ 'master', 'rhel-7.5' ],
```

Comment:
I guess we need to figure out how many of these will we test on each PR.

Reply:
Sorry, I don't understand what you mean? Note that the rhel-7.5 upstream branch doesn't exist yet, but it will at some point, so I thought I already add it. And of course we want to make sure we don't regress master on 7.5

Reply:
I'm fine with this. I am just wondering how many of these rhel versions we want to test on each PR. I know they are important, but too many might become too much of a bottle neck. Anyways we can talk about this later. It doesn't have to block this PR.